internal/task/queue_test.go

```diff
@@ -66,7 +66,7 @@ func TestEnqueueSuccess(t *testing.T) {
 	stopCh := make(chan struct{})
 	// run queue
 	go q.Run(5*time.Second, stopCh)
-	// mock object whichi will be enqueue
+	// mock object which will be enqueue
```

Comment:
This probably should read `enqueued` but it isn't a spelling error (I'm happy to make it if requested)